### PR TITLE
[29_7] remove dead code

### PR DESCRIPTION
**Pull request by tangdouer1005**
In project 29, it is necessary to add conversion methods for pdf format and tm format, but the conversion method from tm to pdf is actually a pdf export method, so the method implemented here is useless

### CODE REVIEW

File: devel/29.tm [hunk 1]
```diff
@@ -57,6 +57,14 @@
       Tangdouer
     </cell>|<\cell>
       Add conversion mechanism between pdf and tm
+    </cell>>|<row|<\cell>
```
Comment (jingkaimori) [inline devel/29.tm:60]: update of project manage document should be submit in separate pr, thus changes to source code can be extracted and ported to upstream texmacs project conveniently.

**Review by jingkaimori — CHANGES_REQUESTED** [review]

File: devel/29.tm (hunk 1 above)
Comment (tangdouer1005) [reply devel/29.tm:60]: ok, i had submitted a doc pr before this. i will modify it
